src/client.ts

```diff
@@ -932,6 +936,10 @@ export class StreamChat<
     if (event.type === 'notification.channel_mutes_updated' && event.me?.channel_mutes) {
       this.mutedChannels = event.me.channel_mutes;
     }
+
+    if (event.type === 'notification.mutes_updated' && event.me?.mutes) {
+      this.mutedUsers = event.me.mutes;
```

Comment:
@ferhatelmas does API always return the whole mutes list or could this be a partial list if the list is long?

Reply:
yes, it cuts by 100